docs/how_to_make_a_tahoe-lafs_release.org

```diff
@@ -1,80 +1,96 @@
-  - [ ] move old release out of ~source/downloads (to downloads/old/?)
-  - [ ] send out relnotes.txt:
-    - add prefix with SHA256 of tarballs, release pubkey, git revhash
-    - GPG-sign the email with release key
-    - send to tahoe-announce@tahoe-lafs.org and tahoe-dev@tahoe-lafs.org
-  - [ ] update Wiki front page: version on download link, News column
-  - [ ] update Wiki "Doc": parade of release notes (with rev of NEWS.rst)
-  - [ ] close the Milestone on the trac Roadmap
-  - [ ] tweet to @tahoelafs
-  - other stuff:
-  - [ ] update https://tahoe-lafs.org/hacktahoelafs/
-  - [ ] make an "announcement of new release" on freshmeat
+     XXX should use full key-id above
+  - [ ] pushed tag to trigger buildslaves: git push official master TAGNAME
+
+* sign the release artifacts [0/8]
+
+  - [ ] pushed signed tag (should trigger Buildbot builders)
+  - [ ] Buildbot workers built all artifacts successfully
+  - [ ] downloaded upstream tarballs+wheels
+  - [ ] announce on IRC that master is unlocked
+  - [ ] compared upstream tarballs+wheels against local copies
+  - [ ] signed each upstream artifacts with "gpg -ba -u 68666a7a FILE"
+     XXX should use full key-id above
+  - [ ] added to relnotes.txt: [0/3]
+    - [ ] prefix with SHA256 of tarballs
+    - [ ] release pubkey
+    - [ ] git revision hash
+  - [ ] GPG-signed the release email with release key (write to relnotes.txt.asc)
+
+* publish release artifacts [0/9]
+
+  - [ ] uploaded to PyPI via: twine upload dist/*
+  - [ ] uploaded *.asc to org ~source/downloads/
+  - [ ] test install works properly: pip install tahoe-lafs
+  - [ ] copied the release tarballs and signatures to tahoe-lafs.org: ~source/downloads/
+  - [ ] moved old release out of ~source/downloads (to downloads/old/?)
+  - [ ] updated readthedocs.org (XXX what does this mean? it updates automatically, no?)
+  - [ ] uploaded wheels to https://tahoe-lafs.org/deps/
+  - [ ] updated txAWS-0.2.1.post5.tar.gz (?)
+  - [ ] uploaded release to https://github.com/tahoe-lafs/tahoe-lafs/releases
+
+* check release downloads [0/]
+
+  - [ ] test PyPI via: pip install tahoe-lafs
+  - [ ] https://github.com/tahoe-lafs/tahoe-lafs/releases
+  - [ ] https://tahoe-lafs.org/downloads/
+  - [ ] https://tahoe-lafs.org/deps/
+
+* document release in trac [0/]
+
+  - [ ] closed the Milestone on the trac Roadmap
+
+* unfreeze master branch [0/]
+
+  - [ ] announced on IRC that new PRs will be looked at/merged
+
+* announce new release [0/]
+
+  - [ ] sent release email and relnotes.txt.asc to tahoe-announce@tahoe-lafs.org
+  - [ ] sent release email and relnotes.txt.asc to tahoe-dev@tahoe-lafs.org
+  - [ ] updated Wiki front page: version on download link, News column
+  - [ ] updated Wiki "Doc": parade of release notes (with rev of NEWS.rst)
+  - [ ] make an "announcement of new release" on freshmeat (XXX still a thing?)
   - [ ] make an "announcement of new release" on launchpad
```

Comment:
I clicked a button to join the Tahoe-LAFS launchpad team.  Someone needs to approve me, I guess.  There haven't been any announcements on Launchpad since 2012 so I'm not sure this is a very important step.  Still, we can try it out if there's anyone around who can still grant the necessary access.

I'm not sure who that would be.  Anyone at all from the Tahoe-LAFS team?  Or Paul Hummer (owner of the Tahoe-LAFS team)?  Not sure I have any current contact information for him.  Maybe Launchpad emailed him when I pushed the join button.